libvita2d/include/vita2d.h

```diff
@@ -69,10 +69,13 @@ void vita2d_draw_line(float x0, float y0, float x1, float y1, unsigned int color
 void vita2d_draw_rectangle(float x, float y, float w, float h, unsigned int color);
 void vita2d_draw_fill_circle(float x, float y, float radius, unsigned int color);
 
-vita2d_texture *vita2d_create_empty_texture(unsigned int w, unsigned int h);
-vita2d_texture *vita2d_create_empty_texture_format(unsigned int w, unsigned int h, SceGxmTextureFormat format);
+vita2d_texture *vita2d_create_empty_texture_advanced(unsigned int w, unsigned int h, SceKernelMemBlockType memtype);
+vita2d_texture *vita2d_create_empty_texture_format_advanced(unsigned int w, unsigned int h, SceGxmTextureFormat format, SceKernelMemBlockType memtype);
 void vita2d_free_texture(vita2d_texture *texture);
 
+#define vita2d_create_empty_texture(x,y) vita2d_create_empty_texture_advanced(x,y,SCE_KERNEL_MEMBLOCK_TYPE_USER_CDRAM_RW)
+#define vita2d_create_empty_texture_format(w,h,x) vita2d_create_empty_texture_format_advanced(w,h,x,SCE_KERNEL_MEMBLOCK_TYPE_USER_CDRAM_RW)
```

Comment:
I would suggest against making these defines, instead being inline functions defined in the header. That avoids a handful of semantic issues, although in all likelihood they'd never come up in the first place.